src/data/mob-name.csv

```diff
@@ -696,3 +697,83 @@
 "シーンマスター","Scene Master"
 "ボンゴエネイヴル","Bongo Enevre"
 "ミサカキッド","Mikasa Kid"
+"シクラメンシャリブ","Cyclamen Charib"
+"オーランピーク","Oran Peak"
+"ハインド","Hind"
+"マインシンバ","Minesimba"
+"デイネルリネーム","Daynelle Rename"
+"インスパイアオビタ","Inspire Ovita"
+"ディミディ","Dimidi"
+"ムギトップランナー","Mugi Top Runner"
+"サタポイント","Sata Point"
+"ハルイロチトセ","Haruiro Chitose"
+"エムティーハリケーン","MT Hurricane"
+"コネクトハット","Connect Hat"
+"キヌゲッコウ","Kinugekko"
+"カズタイフーン","Kazu Typhoon"
+"サンクスステージ","Thanks Stage"
+"イイトヨドライブ","Iitoyo Drive"
+"チャリオスカイ","Chariosky"
+"タツノゼアー","Tatsuno Zeal"
+"ハルイログレート","Haruiro Great"
+"ビンザオウジ","Binza Oji"
+"ハルイログローリー","Haruiro Glory"
+"ブラザーセイテイ","Brother Seitei"
+"グローバルマーズ","Global Mars"
+"フジケンウォッチ","Fujiken Watch"
+"オーソリティダンス","Authority Dance"
+"アマテラビッグ","Amatera Big"
+"ポジショントラップ","Position Trap"
+"トミヤマヴィクトリ","Tomiyama Victory"
+"ライトニングトップ","Lightning Top"
+"デイネルサンダー","Daynelle Thunder"
+"マイノトランセンド","Maino Transcend"
+"トヨナカプレスト","Toyonaka Presto"
+"ヤシュウキング","Yashu King"
+"オーマルセザール","Omar Caesar"
+"クレストカロル","Crest Carol"
+"プリテイスティール","Pretty Steel"
+"アズミパレス","Azumi Palace"
+"ポーラスター","Polar Star"
+"インスパイアケント","Inspier Kent"
+"ルールモデル","Rule Model"
+"アスパオーロ","Aspaolo"
+"レスペデザシンオー","Respedeza Shin O"
+"ピースフルシネマ","Peaceful Cinema"
+"カブトプリンス","Kabuto Prince"
+"レモンネック","Lemon Neck"
+"ガールズギルシャ","Girls Guilsha"
+"ミソハギリバー","Misohagi River"
+"アカマツショック","Akamatsu Shock"
+"ジーズシャープ","Gees Sharp"
+"グローバルスマート","Global Smart"
+"フラッシュフレンド","Flash Friend"
+"ツーリズムキング","Two Rhythm King"
+"グッドスイーツ","Good Sweets"
+"ハイヤードクター","Higher Doctor"
+"ナンソンスルース","Nanson Sleuth"
+"ハンショウワーガー","Hansho Warger"
+"タイジュゴールデン","Taiju Golden"
+"オリエントセブン","Orient Seven"
+"エムティーストーム","MT Storm"
+"アカマツハチマン","Akamatsu Hachiman"
+"プラチナオージ","Platinum OG"
+"ローズナイト","Rose Knight"
+"ベルディアゴーゴー","Verdia Gogo"
+"ミュートハント","Mute Hant"
+"ヒノデフラッグ","Hinode Flag"
+"ヒーローキート","Hero Keat"
+"リメインエルドラ","Remain Eldora"
+"ソニックラン","Sonic Run"
+"ビショクカイタク","Bishoku Kaitaku"
+"ロングヘアのウマ娘","Long-haired Horse Girl"
+"黒鹿毛のウマ娘","Black-haired Horse Girl"
```

Comment:
Can you change this to the coat name to match the others further down?

Reply:
I am on it